package.json

```diff
@@ -15,13 +15,13 @@
   "homepage": "https://desktop.wordpress.com/",
   "main": "build/desktop.js",
   "scripts": {
-    "preinstall": "make check-node-version-parity",
-    "postinstall": "make apply-patches && make rebuild-deps",
+    "preinstall": "make check-node-version-parity && make check-electron-version-parity",
```

Comment:
I saw some other examples that had added these chains (in previous commits) but we don't need to and if we do we can ironically work around what `make` offers.

these changes can go straight into the `Makefile` and rely on `make`'s own dependency-management so that they run in sequence

Reply:
if anything here I would expect us to create a new `make` target like the following

```make
.PHONY: check-version-parity
check-version-parity: check-node-version-parity check-electron-version-parity
```

Reply:
Thanks for the suggestion - agree this is how it really _should_ be done. Fixed in ed8b982.